### PR TITLE
Add BIME Desktop

### PR DESCRIPTION
**Pull request by dlackty**

### CODE REVIEW

File: Casks/bime-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'bime-desktop' do
+  version 'latest'
```
Comment (vitorgalvao) [inline Casks/bime-desktop.rb]: Should be `version :latest`.

**Comment by vitorgalvao** [comment]
It seems the download url is only available behind a registration wall. Is that the case? [If so, it doesn’t belong in this repo](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#unofficial-vendorless-or-walled-builds).

**Comment by dlackty** [comment]
@vitorgalvao Squashed & rebased. 
It's available for everyone: http://www.bimeanalytics.com/download.html

**Comment by vitorgalvao** [comment]
Thank you. Only a small modification needed, then.

File: Casks/bime-desktop.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'bime-desktop' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://s3.amazonaws.com/bimeio/bimedesktop/BimeDesktop-mac.zip'
```
Comment (vitorgalvao) [inline Casks/bime-desktop.rb]: Please add a a `# amazonaws.com is the official download host per the vendor homepage`.

**Comment by dlackty** [comment]
@vitorgalvao Done! Thanks for your help.

**Comment by vitorgalvao** [comment]
Thank you for the submission and corrections. Just noticed something else. It should be `bimedesktop`, not `bime-desktop`.

**Comment by dlackty** [comment]
@vitorgalvao OK. Fixed again! :+1:

**Pull request merged**